src/resources/help/en_US/relnotes3.4.1.html

```diff
@@ -42,6 +42,10 @@
   <li>Upgraded MFEM library version to 4.6.0.</li>
   <li>The <i>surface_normal</i> expression now accepts mesh types other than polydata, allowing it to be used more easily as an operator-created variable.</li>
   <li>Added more keywords ("MFA", "2fa", "Autentication", and "OTP") that cause VisIt clients to open a second popup for a password.</li>
+  <li>GetPlotInformation() can now contain entries for multiple-curves when Query-over-time is performed on multiple variables. For example, if a pick-through-time was performed for variables 'u' and 'v', the curve for 'v' would be retrieved via 'GetPlotInformation()['Curves']['v']. Single variable results will still be retrieved via 'GetPlotInformation()['Curve'].</li>
```

Comment:
```suggestion
```
This entry is already at the bottom of this section.

Reply:
Thanks; I'll fix this!